core/client/peer.js

```diff
@@ -51,6 +51,17 @@ peer = {
         this.closeCall(entry[1].metadata.userId, 0, 'security-user-far');
       });
     }, this.securityCheckInterval);
+
+    // Listen device connection/disconnection to update peers
+    navigator.mediaDevices.addEventListener('devicechange', async () => {
+      const constraints = userStreams.getStreamConstraints(streamTypes.main);
+      constraints.forceNew = true;
```

Comment:
I think we don't need the `forceNew` here. it's useful in the settings to force the creation of a new stream without impacting the one opened with another user